src/main/java/kr/pickple/back/address/dto/response/AllAddressResponse.java

```diff
@@ -0,0 +1,14 @@
+package kr.pickple.back.address.dto.response;
+
+import java.util.List;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
```

Comment:
`p4`

```suggestion
@Getter
@Builder
@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

`@Builder` 어노테이션을 클릭해보면 아래와 같이 나오는데요.

If a member is annotated, it must be either a constructor or a method. If a class is annotated, then a package-private constructor is generated with all fields as arguments (as if `@AllArgsConstructor(access = AccessLevel.PACKAGE`) is present on the class)

클래스 레벨에 `@Builder`를 붙이는 경우 `@AllArgsConstructor`가 생성되면서 access level은 PACKAGE로 설정된다고 합니다. 단순히 `@Builder`만 사용해서 객체를 생성하는 경우 같은 패키지내에서는 new로 생성하지 못하게 access level을 private으로 바꾸는 것은 어떻게 생각하시나요?

Reply:
반영: [3f26347](https://github.com/Java-and-Script/pickple-back/pull/29/commits/3f26347b7e8756030d4882afd14a31e78819b991)
- 이 점을 미처 생각 못했었네요 바로 추가했습니다~